### PR TITLE
additions to cover xhtml files which form part of epub files

### PR DESCRIPTION
**Pull request by TeresaM12**
- added xhtml in ZipDoc.java
- also provided ZipDoc.class (seems to be needed for Win systems)
- amended README.md accordingly

### CODE REVIEW

File: ZipDoc.java [hunk 1]
```diff
@@ -12,6 +12,8 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
+ *
+ * added xhtml files which are part of epub files (*.epub)
```
Comment (costerwi) [inline ZipDoc.java]: There's no need for this comment in the software license.

File: ZipDoc.java [hunk 2]
```diff
@@ -56,7 +58,8 @@ public static void main(final String[] argv) throws IOException, TransformerExce
     @SuppressWarnings("WeakerAccess")
     public static boolean isXml(final String fileName) {
         // TODO Improve this function with a longer list of extensions, or optimally even by inspecting the MIME-type
-        return fileName.endsWith(".xml");
+        //return fileName.endsWith(".xml");
+		return (fileName.endsWith(".xml") ||  fileName.endsWith(".xhtml"));
```
Comment (costerwi) [inline ZipDoc.java]: Thanks for suggestion! You can remove the comment on line 61.

**Comment by TeresaM12** [comment]
I changed both and did a new push. Unfortunately this was obviously not what I should have done to solve the problem. I'm afraid if the new commit cannot be merged I need additional help ...

As a further remark
.epub files might contain .html files instead of .xhtml files. I do not know the java class (    SAXTransformerFactory). If it also provides nice output for html files in general maybe html can be added as well.

**Review by costerwi — APPROVED** [review]
Thanks for your contribution!

**Pull request merged**